readium/shared/src/main/java/org/readium/r2/shared/util/Language.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 Readium Foundation. All rights reserved.
+ * Use of this source code is governed by the BSD-style license
+ * available in the top-level LICENSE file of the project.
+ */
+
+package org.readium.r2.shared.util
+
+import java.util.*
+
+class Language private constructor(val code: String, val locale: Locale) {
```

Comment:
I introduced this `Language` to have a higher language construct that doesn't depend on the JVM, and to add some useful helpers such as `removeRegion()`.